index.html

```diff
@@ -1,12 +1,216 @@
+        <img
+          id="hamburger"
+          class="hamburger"
+          src="./assets/img/mobile-menu-1.png"
+          alt=""
+        />
+        <!-- LOGO -->
+        <a href="index.html">
+          <img class="logo" src="./assets/img/logo.png" alt="" />
+        </a>
+        <ul>
+          <a href="about.html" class="text-14-bold">About</a>
+          <a href="#" class="text-14-bold">Program</a>
+          <a href="#" class="text-14-bold">Join</a>
+          <a href="#" class="text-14-bold">Sponsor</a>
+          <a href="#" class="text-14-bold">News</a>
+          <!-- Action button -->
+          <a href="#" class="action-btn-1 text-14-bold">WD Campaign</a>
+        </ul>
+      </nav>
+    </header>
+    <!-- Hero section -->
+    <section class="hero-container flex-column">
+      <div class="hero flex-column">
+        <div class="hero-title">
+        </div>
+        <p class="sub-theme flex-row text-12 big-text-20 custom-gray-2">
+          A joyful celebration believing in the value of openness and sharing,
+          creating a positive change with people from over 80 countries is
+          taking place in October, in Korea.
+        </p>
+        <div class="date-venue flex-column">
+          <h2 class="custom-gray-3 text-20-bold big-text-42">2023.10.19(THU) ~ 20(FRI)</h2>
+          <p class="custom-gray-4 text-13 big-text-27">
+            @ National Museum of Karea, Art Center Nabi and more
+          </p>
+        </div>
+      </div>
+    </section>
+    <!-- Program section-->
+    <section class="program-container">
+      <div class="program flex-column">
+        <div class="title flex-column">
+          <h4 class="custom-white text-20 flex-row">Main Program</h4>
+          <div class="title-underline"></div>
+        </div>
+        <div class="program-details">
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/lecture-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">Lecture</h3>
+              <p class="custom-white text-10 big-text-12">
+                Listen to the speakers from various countries about the messages
+                of sharing and opening.
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/exhibition-icon.png" />
+            </div>
+            <div class="program-card-texts">
+              <h3 class="custom-red-2 text-14 big-text-20">CC Exhibition</h3>
+              <p class="custom-white text-10 big-text-12">
+                Appreciate various creations applying CC license of artists,
+                organized from Art Center Nabi
+              </p>
+            </div>
+          </div>
+          <div class="program-card">
+            <div class="icon-box">
+              <img src="./assets/img/forum-icon.png" />
```

Comment:
[REQUIRED] Please include the alt attribute and a value in all image tags that are not just meant for styling or design. But add an empty string as the value if the image is meant for styling and beautification.